### ISSUE TITLE
Add Java F3 screen

### ISSUE THREAD

**Issue by YHDiamond** [GeyserMC/GeyserOptionalPack#15]
**What feature do you want?**
A command like "/geyser f3" which would show all the debug information that you get on Java when you press f3 (preferably in a similar looking GUI).

**Alternatives?**
There are coordinates that can be enabled or disabled, but that doesn't contain things like block looking at, or biome. Theoretically certain options can be added using plugins and be displayed in chat or with a toolbar, but that won't show everything in one place.

**How I would recommend going about adding this** 
@notnotnotswipez and I spoke about this a lot, thinking that a possible way would be to make a custom resource pack with MoLang that would make a request from the Geyser server for information. This pack would be applied on join. Depending on your stored options in Geyser, the request would either return empty, as to not interfere with the Bedrock player's screen, or if they have f3 enabled, the request would contain all the necessary information.

**Comment by RednedEpic** [comment]
Transferred to GeyserOptionalPack as this is a better place for it.

**Comment by YHDiamond** [comment]
👍

**Comment by Kas-tle** [comment]
@notnotnotswipez 
I would look into implementing the toggle with subpacks. See https://wiki.bedrock.dev/documentation/subpacks.

**Comment by TBYT** [comment]
this issue still open? The spigot plugin https://github.com/Heath123/debuginfo-be was updated, and it seems YHDiamond you have someone working on it, unless this issue is open just for anyone to add any f3 features to the optional pack?

**Comment by Jens-Co** [comment]
Correct, this issue/feature request will stay open until dev adds such mechanics into the optional repo or the feature will never be added.
